src/migration/file-migration-provider.ts

```diff
@@ -31,11 +31,13 @@ export class FileMigrationProvider implements MigrationProvider {
     for (const fileName of files) {
       if (
         fileName.endsWith('.js') ||
+        fileName.endsWith('.cjs') ||
         (fileName.endsWith('.ts') && !fileName.endsWith('.d.ts')) ||
         fileName.endsWith('.mjs') ||
         (fileName.endsWith('.mts') && !fileName.endsWith('.d.mts'))
       ) {
         const migration = await import(
+            this.#props.protocol +
```

Comment:
This is now `undefined` by default and will break everything.

Reply:
sorry 🙏 
did this qwickly from work on the github vscode web interface 
this should fix it

i would prefer rather this solution than this ?

```ts
const isEsm = Boolean(typeof module !== 'undefined' && typeof module.imports !== 'undefined') 
this.#props.path.join(
  isEsm ? "file://" : "",
  this.#props.migrationFolder,
  fileName
)
```